docs/release-notes/1.0.0/release-1.0.0.md

```diff
@@ -0,0 +1,80 @@
+# ML.NET 1.0.0 Release Notes
+
+[ML.NET](https://aka.ms/mlnet) is now 1. :cake:. After 12 preview releases over the last year and implementing many enhancements and API improvements, this release brings you the version `1.0.0` of [ML.NET](https://aka.ms/mlnet). Thank you wholeheartedly for being an awesome comunity. Please continue to use and provide feedback on what should be improved. We will ensure that going forward no breaking API changes are introduced.
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more details.
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
```

Comment:
Anything new since the last release note?  Perhaps a list of bugs and enhancement?  What about docs update

Reply:
We decided not to have that here and work on it for the blog posts. It is mostly documentation BTW.